lib/models/usuario.js

```diff
@@ -76,6 +80,64 @@ export default class Usuario extends Model {
     return omit(['contrasenia'], super.toJSON());
   }
 
+  async getTurnos() {
+    const turnos = await Turno.findAll({
+      where: {
+        usuarioId: this.id,
+      },
+      include: [
+        {
+          model: Actividad,
+          attributes: [
+            'id',
+            'nombre',
+            'fechaHoraInicio',
+            'fechaHoraFin',
+            'responsable',
+          ],
+          include: [
+            {
+              model: Espacio,
+              attributes: ['nombre'],
+              include: [
+                {
+                  model: Edificio,
+                  attributes: ['nombre'],
+                },
+              ],
+            },
+          ],
+        },
+      ],
+    });
+
+    return turnos;
+  }
+
+  async getIdsCarreras() {
+    const inscripciones = await this.getInscripcionCarreras({
+      attributes: ['idCarrera'],
+      where: { usuarioId: this.id },
+    });
+
+    return pluck(['idCarrera'], inscripciones);
+  }
+
+  async getIdsActividadesConTurno() {
+    const ids = await Turno.findAll({
+      where: {
+        usuarioId: this.id,
+      },
+      attributes: [],
+      include: {
+        model: Actividad,
+        attributes: ['id'],
+      },
+    });
```

Comment:
Por lo mismo que te dije más arriba:

```suggestion
    const ids = await this.getTurnos(
      attributes: [],
      include: {
        model: Actividad,
        attributes: ['id'],
      },
      raw: true
    });
```